examples/grpc-kotlin/src/main/kotlin/example/armeria/grpc/kotlin/HelloServiceImpl.kt

```diff
@@ -42,13 +38,12 @@ class HelloServiceImpl : HelloServiceGrpc.HelloServiceImplBase() {
         // 3. Call a blocking API in the separate thread pool you manage.
         //
         // In this example, we chose the option 1:
```

Comment:
This comment seems inconsistent with the actual code

Reply:
Let me remove all of this comment. Because this method does not run in the event loop.
I am investigating how to run a coroutine on an Armeria event loop.